google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ConnectionAsyncApiTest.java

```diff
@@ -390,18 +397,6 @@ public void testExecuteClientSideQueryAsync() {
     }
   }
 
-  @Test
-  public void testExecuteInvalidQueryAsync() {
```

Comment:
Why is this test case removed? If I understand it correctly, this would execute a DML statement without a returning clause, which means that it should still fail.

Reply:
Reinstated the test with small change in when the exception will be raised.